src/Stackdriver.php

```diff
@@ -61,6 +62,10 @@ public function startTracing(): void
             return;
         }
 
+        if (Str::startsWith(request()->getPathInfo(), config('stackdriver.tracing.blacklist'))) {
```

Comment:
This does not allow people to exclude paths based on patterns. I prefer Telescope's approach:
https://github.com/laravel/telescope/blob/53c40e89f0de45d878665f17f2da02a05c18a394/src/Telescope.php#L180-L188

Something like this:
``` php
$blacklist = config('stackdriver.tracing.blacklist', []);
if ($this->app['request']->is($blacklist)) {
    return;
}
```

Reply:
Great, done!